app/controller/dashboard.js

```diff
@@ -1,11 +1,11 @@
 import Hubkit from 'hubkit';
 import Promise from 'bluebird';
-import _ from 'underscore';
+import _ from 'lodash';
```

Comment:
This was from when I was working on some advanced configuration stuff. It could be changed back, but from what I have read, lodash has some performance improvements from underscore, and included most and then some of underscores functions.